.github/workflows/build_all_targets.yml

```diff
@@ -238,5 +238,7 @@ jobs:
         if: startsWith(github.ref, 'refs/tags/v')
         with:
           draft: true
-          files: artifacts/*.px4
+          files: |
+            artifacts/*.px4
+            artifacts/cannode/**/*.uavcan.bin
```

Comment:
Do you also need to upload to S3?

Reply:
Also, this won't create a `cannode` directory in the release assets, it will only upload the `*.uavcan.*bin` files next to the `*.px4` files